### PR TITLE
ACS-6736: Add unit tests of BulkIngesterEventProcessor

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/bulk_ingester/BulkIngesterEventProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+{
+    private static final String NODE_ID = "07659d13-8d64-4905-a329-6b27fe182023";
+    private static final String NODE_TYPE = "cm:folder";
+    private static final String CREATOR_ID = "admin";
+    private static final String MODIFIER_ID = "hr_user";
+    private static final Set<String> ASPECT_NAMES = Set.of("cm:titled");
+    private static final long CREATED_AT = 1000L;
+    private final IngestMetadataCommandHandler ingestMetadataCommandHandler = mock();
+    private final IngestContentCommandHandler ingestContentCommandHandler = mock();
+    private final BulkIngesterEventProcessor bulkIngesterEventProcessor = new BulkIngesterEventProcessor(
```
Comment (tpage-alfresco) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/bulk_ingester/BulkIngesterEventProcessorTest.java:58]: FWIW you can use `@Mock` and `@InjectMocks` to wire up the mocks. I think it makes it very slightly easier to see which the class under test is, but this is fine too.

**Review by tpage-alfresco — APPROVED** [review]
LGTM

**Review by krdabrowski — APPROVED** [review]
LGTM

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/bulk_ingester/BulkIngesterEventProcessorTest.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/bulk_ingester/BulkIngesterEventProcessorTest.java]: done!

**Pull request merged**